processor.go

```diff
@@ -140,40 +140,38 @@ func NewProcessor(protoReader io.Reader) (*Processor, error) {
 	}, nil
 }
 
+type symListElem struct {
+	symbol string
+	// noMerge means this is a user-defined symbol we're not allowed to merge with neighbors.
+	noMerge bool
+}
+
+func (e symListElem) Dead() bool {
```

Comment:
I don't love this commit... since it doesn't affect run-time, let's back it out for now. I'd like to think a bit more about memory management in Encode in general

Reply:
OK

Reply:
Done